libretro-common/include/string/stdstring.h

```diff
@@ -149,8 +149,10 @@ char *string_trim_whitespace_right(char *const s);
 char *string_trim_whitespace(char *const s);
 
 /* max_lines == 0 means no limit */
-void word_wrap(char *dst, const char *src,
-      int line_width, bool unicode, unsigned max_lines);
+void word_wrap(char *dst, size_t dst_size, const char *src,
+      int line_width, int wideglyph_width, unsigned max_lines);
+void word_wrap_wideglyph(char *dst, size_t dst_size, const char *src,
+      int line_width, int wideglyph_width, unsigned max_lines);
 
```

Comment:
Could you please add comments here explaining the meaning of the `wideglyph_width` argument?

I understand why you have done it like this (with a base value of 100 - more efficient than using a float factor), but the usage of core functions like this must be entirely transparent. Any developer who comes along must be able to identify the correct usage of these word wrap implementations without having to scan the underlying source code.